src/attestation/azext_attestation/tests/latest/test_attestation_scenario.py

```diff
@@ -100,8 +93,7 @@ def call_scenario(test, rg):
     setup(test, rg)
     step_attestationproviders_create(test, rg)
     step_attestationproviders_get(test, rg)
-    step_attestationproviders_list(test, rg)
-    step_attestationproviders_listbyresourcegroup(test, rg)
+    # step_attestationproviders_listbyresourcegroup(test, rg) Service Internal Error
     step_attestationproviders_delete(test, rg)
     cleanup(test, rg)
 
```

Comment:
I don't find the tests for data-plane operations.

Reply:
> I don't find the tests for data-plane operations.

Data-plane tests were just added. Please check it out again. Thanks.